src/lib/run_environment.ml

```diff
@@ -28,10 +28,11 @@ module Tool = struct
       [@@@ocaml.warning "-11"]
       type t = [
         | `Bwa of [ `V_0_7_10 ]
-        | `Custom of string * string
+        (* A tool that is manually installed by retrieving source or binary.*)
```

Comment:
I wasn't certain of this. `opam` as a tool doesn't seem that much different than all of the other stuff defined as `custom`. I couldn't figure out what made `bwa` special.